diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index 14f0fdf..a5cc31a 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -570,7 +570,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI__default['default'] || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit__default['default'](40);
+const npmRequestLimit = pLimit__default['default'](3);
 const npmPublishLimit = pLimit__default['default'](10);
 
 function jsonParse(input) {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 1c227ab..b84e188 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -547,7 +547,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI$1 || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
