diff --git a/node_modules/@graphql-tools/delegate/dist/index.cjs b/node_modules/@graphql-tools/delegate/dist/index.cjs
index c569eb4..2688470 100644
--- a/node_modules/@graphql-tools/delegate/dist/index.cjs
+++ b/node_modules/@graphql-tools/delegate/dist/index.cjs
@@ -3075,7 +3075,7 @@ function delegateToSchema(options) {
     }
   }
   const fragments = info ? getFragmentDefinitions(info) : void 0;
-  const targetRootType = operation === graphql.OperationTypeNode.MUTATION ? targetSchema.getMutationType() : operation === graphql.OperationTypeNode.SUBSCRIPTION ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
+  const targetRootType = operation === "mutation" ? targetSchema.getMutationType() : operation === "subscription" ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
   const request = createRequest({
     subgraphName: schema.name,
     fragments,
diff --git a/node_modules/@graphql-tools/delegate/dist/index.js b/node_modules/@graphql-tools/delegate/dist/index.js
index 58f0115..95a02cb 100644
--- a/node_modules/@graphql-tools/delegate/dist/index.js
+++ b/node_modules/@graphql-tools/delegate/dist/index.js
@@ -1,6 +1,6 @@
 import { memoize2, memoize1, collectFields, relocatedError, mergeDeep, promiseReduce, pathToArray, memoize3, getDefinedRootType, createGraphQLError, implementsAbstractType, getRootTypeNames, astFromArg, astFromValueUntyped, asArray, isAsyncIterable, getOperationASTFromRequest } from '@graphql-tools/utils';
 export { createDeferred } from '@graphql-tools/utils';
-import { GraphQLError, locatedError, isAbstractType, getNullableType, isLeafType, isCompositeType, isListType, responsePathAsArray, Kind, TypeInfo, versionInfo, visit, visitWithTypeInfo, isNullableType, isUnionType, getNamedType, isObjectType, isInterfaceType, typeFromAST, isTypeSubTypeOf, isNonNullType, isInputObjectType, defaultFieldResolver, isSchema, OperationTypeNode, validate } from 'graphql';
+import { GraphQLError, locatedError, isAbstractType, getNullableType, isLeafType, isCompositeType, isListType, responsePathAsArray, Kind, TypeInfo, versionInfo, visit, visitWithTypeInfo, isNullableType, isUnionType, getNamedType, isObjectType, isInterfaceType, typeFromAST, isTypeSubTypeOf, isNonNullType, isInputObjectType, defaultFieldResolver, isSchema, validate } from 'graphql';
 import { handleMaybePromise, isPromise, createDeferredPromise, mapAsyncIterator } from '@whatwg-node/promise-helpers';
 import { CRITICAL_ERROR, executorFromSchema } from '@graphql-tools/executor';
 export { executorFromSchema as createDefaultExecutor } from '@graphql-tools/executor';
@@ -3075,7 +3075,7 @@ function delegateToSchema(options) {
     }
   }
   const fragments = info ? getFragmentDefinitions(info) : void 0;
-  const targetRootType = operation === OperationTypeNode.MUTATION ? targetSchema.getMutationType() : operation === OperationTypeNode.SUBSCRIPTION ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
+  const targetRootType = operation === "mutation" ? targetSchema.getMutationType() : operation === "subscription" ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
   const request = createRequest({
     subgraphName: schema.name,
     fragments,
