diff --git a/node_modules/@graphql-tools/delegate/dist/index.cjs b/node_modules/@graphql-tools/delegate/dist/index.cjs
index 41bef7b..7eb5239 100644
--- a/node_modules/@graphql-tools/delegate/dist/index.cjs
+++ b/node_modules/@graphql-tools/delegate/dist/index.cjs
@@ -3013,7 +3013,7 @@ function delegateToSchema(options) {
     }
   }
   const fragments = info ? getFragmentDefinitions(info) : void 0;
-  const targetRootType = operation === graphql.OperationTypeNode.MUTATION ? targetSchema.getMutationType() : operation === graphql.OperationTypeNode.SUBSCRIPTION ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
+  const targetRootType = operation === "mutation" ? targetSchema.getMutationType() : operation === "subscription" ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
   const request = createRequest({
     subgraphName: schema.name,
     fragments,
diff --git a/node_modules/@graphql-tools/delegate/dist/index.js b/node_modules/@graphql-tools/delegate/dist/index.js
index 869e6fa..256f81e 100644
--- a/node_modules/@graphql-tools/delegate/dist/index.js
+++ b/node_modules/@graphql-tools/delegate/dist/index.js
@@ -3013,7 +3013,7 @@ function delegateToSchema(options) {
     }
   }
   const fragments = info ? getFragmentDefinitions(info) : void 0;
-  const targetRootType = operation === OperationTypeNode.MUTATION ? targetSchema.getMutationType() : operation === OperationTypeNode.SUBSCRIPTION ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
+  const targetRootType = operation === "mutation" ? targetSchema.getMutationType() : operation === "subscription" ? targetSchema.getSubscriptionType() : targetSchema.getQueryType();
   const request = createRequest({
     subgraphName: schema.name,
     fragments,
