diff --git a/node_modules/@guild-docs/client/NextNProgress.js b/node_modules/@guild-docs/client/NextNProgress.js
index b83f7cf..658a710 100644
--- a/node_modules/@guild-docs/client/NextNProgress.js
+++ b/node_modules/@guild-docs/client/NextNProgress.js
@@ -3,7 +3,9 @@ import RouterImport from 'next/router.js';
 import NProgress from 'nprogress';
 import React, { memo, useMemo, useEffect } from 'react';
 import { getDefault } from './utils.js';
-import { useLatestRef } from '@chakra-ui/react';
+import chakraUiReact from '@chakra-ui/react';
+
+const { useLatestRef } = chakraUiReact;
 
 const Router = getDefault(RouterImport);
 let startTimer = null;
