diff --git a/node_modules/@guild-docs/mdx-remote/index.js b/node_modules/@guild-docs/mdx-remote/index.js
index 8571c8e..bc1f48d 100644
--- a/node_modules/@guild-docs/mdx-remote/index.js
+++ b/node_modules/@guild-docs/mdx-remote/index.js
@@ -1,6 +1,6 @@
 import * as mdx from '@mdx-js/react';
 import { useState, useEffect, useMemo, createElement } from 'react';
-import * as runtime from 'react/jsx-runtime.js';
+import * as runtime from 'react/jsx-runtime';
 
 function MDXRemote({ compiledSource, scope, components = {}, lazy, frontmatter }) {
   const [isReadyToRender, setIsReadyToRender] = useState(!lazy || typeof window === "undefined");
