diff --git a/node_modules/eslint-plugin-n/lib/index.js b/node_modules/eslint-plugin-n/lib/index.js
index a8e8ce3..4f16b10 100644
--- a/node_modules/eslint-plugin-n/lib/index.js
+++ b/node_modules/eslint-plugin-n/lib/index.js
@@ -45,6 +45,10 @@ const configs = {
 }
 
 /** @type {ESLint.Plugin & { configs: Configs }} */
+export const meta = base.meta
+export const rules = base.rules
+export { configs }
+
 const plugin = Object.assign(base, { configs })
 
 export {
