diff --git a/node_modules/graphql-ws/dist/use/ws.d.ts b/node_modules/graphql-ws/dist/use/ws.d.ts
index 60a0262..cd7bac0 100644
--- a/node_modules/graphql-ws/dist/use/ws.d.ts
+++ b/node_modules/graphql-ws/dist/use/ws.d.ts
@@ -5,7 +5,7 @@ import { b as ConnectionInitMessage, a as Disposable } from '../common-DY-PBNYy.
 import { S as ServerOptions } from '../server-CvxoLxzz.js';
 import 'graphql';
 
-type WebSocketServer = WebSocket.Server;
+type WebSocketServer = import('ws').Server;
 
 /**
  * The extra that will be put in the `Context`.
